cycode/cli/code_scanner.py

```diff
@@ -1099,6 +1112,21 @@ def _is_file_extension_supported(scan_type: str, filename: str) -> bool:
     return not filename.endswith(consts.SECRET_SCAN_FILE_EXTENSIONS_TO_IGNORE)
 
 
+def _is_iac(scan_type: str) -> bool:
+    return scan_type == consts.INFRA_CONFIGURATION_SCAN_TYPE
+
+
+def _is_tfplan_json_file(file: str, content: str) -> bool:
+    if not file.endswith('.json'):
+        return False
+    try:
+        tf_plan = json.loads(content)
+        return tf_plan.get('resource_changes')
+
+    except ValueError:
```

Comment:
`json.JSONDecodeError` instead? to be sure that we are handling json errors properly only and not something too general